ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/endpoint/StopStrategies2.java

```diff
@@ -0,0 +1,51 @@
+package com.sequenceiq.cloudbreak.ccmimpl.endpoint;
+
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.concurrent.Immutable;
+
+import com.github.rholder.retry.Attempt;
+import com.github.rholder.retry.StopStrategies;
+import com.github.rholder.retry.StopStrategy;
+
+/**
+ * Factory class for instances of {@link StopStrategy}.
+ */
+public final class StopStrategies2 {
```

Comment:
It was named this way because it was more strategies like the ones in StopStrategies in the library, and because this is the general naming convention in Director for utility classes. But I have renamed it according to your request.